### ISSUE TITLE
ML: VGG16 Model #1 (24 Layers): Configuration#3: Training model and store the results in the directory for 30 and 50 epochs

### ISSUE THREAD

**Issue by hmehryar** [hmehryar/Hm.JetscapeMl#267]
Model Architect: 

- 4Conv(Conv+Conv+MaxPool+Dropout)
- 4Dense(Dense+Dropout)

1. Put the training on the Grid by using jetscape-ml-tensorflow-cnn.sh
2. save the model and the evaluation data in the power point [slides](https://docs.google.com/presentation/d/1H7D-TKbmZjwNFNcf8Ecuz3HLl1veCZVL/edit?usp=sharing&ouid=108370064464811889896&rtpof=true&sd=true)
3. update the diagrams data on the power point slides

**Comment by hmehryar** [comment]
30 epochs

839936/840000 [============================>.] - ETA: 0s - loss: 4.5717 - accuracy: 0.9317
Epoch 00012: val_accuracy did not improve from 0.84683

840000/840000 [==============================] - 308s 367us/sample - loss: 4.5717 - accuracy: 0.9317 - val_loss: 4.5195 - val_accuracy: 0.8195
Epoch 00012: early stopping

**Comment by hmehryar** [comment]
50 epochs 
Epoch 31/50
839936/840000 [============================>.] - ETA: 0s - loss: 0.6477 - accuracy: 0.9345  
Epoch 00031: val_accuracy did not improve from 0.94070
840000/840000 [==============================] - 316s 376us/sample - loss: 0.6477 - accuracy: 0.9345 - val_loss: 0.6503 - val_accuracy: 0.9238
Epoch 00031: early stopping

**Comment by hmehryar** [comment]
30 epochs

Epoch 12/50
839936/840000 [============================>.] - ETA: 0s - loss: 4.5717 - accuracy: 0.9317  
Epoch 00012: val_accuracy did not improve from 0.84683
840000/840000 [==============================] - 308s 367us/sample - loss: 4.5717 - accuracy: 0.9317 - val_loss: 4.5195 - val_accuracy: 0.8195
Epoch 00012: early stopping

**Comment by hmehryar** [comment]
port to excel for overleaf
